client/src/components/ProfileListings/ProfileListings.tsx

```diff
@@ -0,0 +1,87 @@
+import { Grid } from '@mui/material';
+import UserCard from '../UserCard/UserCard';
+
+const ProfileListings = () => {
+  return (
+    <Grid container rowSpacing={4} sx={{ padding: '0 5vw' }}>
+      {dummyData.map((user) => (
+        <Grid item display="flex" justifyContent="center" xs={12} sm={6} md={4} key={user.name}>
```

Comment:
Make sure to add. `key` prop onto the Grid item.

Reply:
There is a key prop at the end of line 8

Reply:
Oh yes - didn't see it there!